.github/workflows/main.yml

```diff
@@ -22,7 +22,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        java_version: ['8', '11', '17', '18']
+        java_version: ['8', '11', '17']
```

Comment:
addons:
  apt:
    packages:
      - openjdk-18-jdk

Reply:
where does this go - github actions yaml is really badly documented - I can't find any examples like this to copy from

Reply:
oh shouldn't use it as a java version direclty, as it is an apt install on the o.s. if applicable :) I think you're right, perhaps better to wait for travis to officially have an image available, testing it now it also needs the apt set-18-default apt as well,